src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -1,353 +0,0 @@
-                if (ScriptSettingsManager.Instance.GetSetting(EnvironmentSettingNames.AzureWebsitePlaceholderMode) == "1")
-                {
-                    return true;
-                }
-
-                // no longer standby mode
-                _standbyMode = false;
-
-                return _standbyMode.Value;
-            }
-        }
-
-        /// <summary>
-        /// Ensures that the host has been fully initialized and startup
-        /// has been initiated. This method is idempotent.
-        /// </summary>
-        public Task EnsureHostStarted(CancellationToken cancellationToken)
-        {
-            if (!_hostStarted)
-            {
-                lock (_syncLock)
-                {
-                    if (!_hostStarted)
-                    {
-                        _runTask = Task.Run(() => RunAndBlock(cancellationToken));
-                        _hostStarted = true;
-                    }
-                }
-            }
-
-            return _runTask;
-        }
-
-        protected override void Dispose(bool disposing)
-        {
-            if (disposing)
-            {
-                (_secretManager as IDisposable)?.Dispose();
-
-                _metricsLogger?.Dispose();
-            }
-
-            base.Dispose(disposing);
-        }
-
-        // this is for testing only
-        internal static void ResetStandbyMode()
-        {
-            _standbyMode = null;
-        }
-
-        protected override void OnInitializeConfig(ScriptHostConfiguration config)
-        {
-            base.OnInitializeConfig(config);
-
-            // Note: this method can be called many times for the same ScriptHostConfiguration
-            // so no changes should be made to the configuration itself. It is safe to modify
-            // ScriptHostConfiguration.Host config though, since the inner JobHostConfiguration
-            // is created on each restart.
-
-            // Add our WebHost specific services
-            var hostConfig = config.HostConfig;
-
-            hostConfig.AddService<IMetricsLogger>(_metricsLogger);
-            hostConfig.AddService<IWebHookProvider>(_bindingWebHookProvider);
-
-            // Add our exception handler
-            hostConfig.AddService<IWebJobsExceptionHandler>(_exceptionHandler);
-
-            // HostId may be missing in local test scenarios.
-            var hostId = hostConfig.HostId ?? "default";
-            Func<string, FunctionDescriptor> funcLookup = (name) => this.Instance.GetFunctionOrNull(name);
-            var loggingConnectionString = config.HostConfig.DashboardConnectionString;
-
-            // TODO: This is asking for a LoggerFactory before the LoggerFactory is ready. Pass a Null instance for now.
-            var instanceLogger = new FunctionInstanceLogger(funcLookup, _metricsLogger, hostId, loggingConnectionString, NullLoggerFactory.Instance);
-            hostConfig.AddService<IAsyncCollector<FunctionInstanceLogEntry>>(instanceLogger);
```

Comment:
Where is this FunctionInstanceLogger registered now? As you know, this is how our important ETW events ultimately get raised at the right times in the invocation pipeline.